docs/howtos/exposing-rust-components-to-kotlin.md

```diff
@@ -0,0 +1,203 @@
+
+## How to expose your Rust Component to Kotlin
+
+Welcome!
+
+It's great
```

Comment:
This goes for a lot of the text, but I'd prefer if these things were justified to a larger column width. It's a little hard to read as is.

I use https://marketplace.visualstudio.com/items?itemName=stkb.rewrap to do this more automatically frequently, there are equivalent tools for basically every text editor I've ever used though.